packages/ember-glimmer/tests/integration/components/target-action-test.js

```diff
@@ -259,6 +260,34 @@ moduleFor('Components test: sendAction', class extends RenderingTest {
 
     this.runTask(() => innerChild.sendAction('bar', 'something special'));
   }
+
+  ['@test asserts if called on a destroyed component']() {
+    let component;
+
+    this.registerComponent('rip-alley', {
+      ComponentClass: Component.extend({
+        init() {
+          this._super();
+          component = this;
+        },
+
+        toString() {
+          return 'component:rip-alley';
+        }
+      })
+    });
+
+    this.render('{{rip-alley}}');
+
+    runDestroy(component);
```

Comment:
I’d vaguely prefer a more natural way to destroy the component, e.g. render it in an `if` and toggle the condition 

What do you think?

Reply:
@rwjblue didn't think of that, but I agree sticking closer to real world is best. Will update to that. Do we have a helper or do I need to wrap a set in a run?